packages/fast-components-react-base/src/tabs/tabs.tsx

```diff
@@ -142,7 +142,7 @@ class Tabs extends Foundation<ITabsHandledProps & ITabsManagedClasses, ITabsUnha
                     "aria-selected": this.state.activeId === tabItem.props.id,
                     onClick: this.handleClick,
                     onKeyDown: this.handleKeyDown,
-                    tabIndex: this.state.activeId !== tabItem.props.id ? -1 : 0
+                    tabIndex: this.state.activeId !== tabItem.props.id ? -1 : 1
```

Comment:
Should be 0 instead of 1.

Reply:
@eljefe223 to add to this it's helpful to look at https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex which is a more accurate source for this kind of information.